ops/ansible/playbooks-ccs/vars/prod-sbx/group_vars/all/env_specific.yml

```diff
@@ -1,42 +1,51 @@
 ---
-
+  
 # Version of the ETL Jar to deploy 
 data_pipeline_version: '0.1.0-SNAPSHOT'
 
 # Bucket where deployment artifacts are located. 
 s3_bucket_deployment: 'bfd-mgmt-artifacts-577373831711'
 
 # Name of ETL Staging Bucket 
-s3_bucket_etl_staging: 'bfd-prod-sbx-etl-577373831711'
-
-# The abbreviated name for this environment, per the naming conventions used by HealthAPT.
-# env_name: 'ts'
+s3_bucket_etl_staging: 'bfd-prod-etl-577373831711'
```

Comment:
Why this change for the ETL staging bucket? Why use a `prod` named bucket in the prod-sbx vars?

Reply:
fixed. thx.